### ISSUE TITLE
Local installations now require gcc

### ISSUE THREAD

**Issue by czlee** [czlee/tabbycat#503]
django-gulp requires psutil:
https://github.com/beaugunderson/django-gulp/blob/master/setup.py#L25
and psutil requires gcc:
https://github.com/giampaolo/psutil/blob/master/INSTALL.rst#linux

This means we need to do one of two things:
- Drop django-gulp from our dependencies, or
- Add gcc to the list of things users have to install as part of a local installation

Installing gcc shouldn't be a big deal for anyone who deals with command-line tools often—lots of things require it. But it might be an issue for some members of our target audience, even for local installations. I'm not totally opposed to just requiring gcc, but maybe there's a way around it.

**Comment by czlee** [comment]
To add to my claim that this isn't _that_ big a deal, all things considered: Remember that we already require `python3-dev` in local installations, which is a package targeted specifically at developers.

**Comment by philipbelesky** [comment]
I've removed django_gulp as a require dependency as its only really necessary as a development aid. If the front end files are not changing its not necessary to constantly recompile them. I've modified the install instructions and testing on Windows/OS X/heroku.

Note that django_gulp is still available as a development dependency, and has (like debug_toolbar) an explicit flag in local_settings that allows it to be used if necessary.
